lighthouse-core/computed/metrics/lantern-largest-contentful-paint.js

```diff
@@ -65,6 +74,21 @@ class LanternLargestContentfulPaint extends LanternMetric {
     );
   }
 
+  /**
+   * @param {LH.Gatherer.Simulation.Result} simulationResult
+   * @param {any=} extras
+   * @return {LH.Gatherer.Simulation.Result}
+   */
+  static getEstimateFromSimulation(simulationResult, extras) { // eslint-disable-line no-unused-vars
```

Comment:
any reason to keep `extras` around if not used?

(if it's good to keep around, should be typed after #10461)

Reply:
not particularly other than reminder that you *can* use it, but that's pretty weak